SYCL/Plugin/interop-level-zero-interop-task-mem.cpp

```diff
@@ -0,0 +1,65 @@
+// REQUIRES: level_zero, level_zero_dev_kit
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// Test for Level Zero interop_task.
+
+#include <CL/sycl.hpp>
+// clang-format off
+#include <level_zero/ze_api.h>
+#include <CL/sycl/backend/level_zero.hpp>
+// clang-format on
+
+#define SIZE 16
+
+class my_selector : public cl::sycl::device_selector {
```

Comment:
```suggestion
class level_zero_gpu_selector : public sycl::device_selector {
```
I don't think this custom device selector is really needed. 

// REQUIRES: level_zero, level_zero_dev_kit
// RUN: %GPU_RUN_PLACEHOLDER %t.out

I think this setting expose only level_zero gpu to the test program.